transformers4rec/torch/model/base.py

```diff
@@ -545,12 +554,19 @@ def __init__(
         self.optimizer = optimizer
         self.hf_format = hf_format
```

Comment:
Can we remove setting the `hf_format` from the model's init method? (from the model's forward method and the docstring as well) This was used to set the `hf_format` of `NextItemPredictionTask` to False after training to prepare the model for inference. But with the new refactoring introduced in this PR, the model will always return a tensor at inference (which is also the default mode when calling `model(inputs)`)